src/Core/src/Handlers/Image/ImageHandler.Android.cs

```diff
@@ -62,14 +62,12 @@ await handler
 
 		public override void PlatformArrange(Graphics.Rect frame)
 		{
-			if (PlatformView.GetScaleType() == ImageView.ScaleType.CenterCrop)
+			if (PlatformInterop.IsImageViewCenterCrop(PlatformView))
```

Comment:
Ok, I fear the original `if` block *never* worked and was always `false`?!?

Once I ported to Java, it goes in the `if` successfully?!?